README.md

```diff
@@ -161,6 +161,16 @@ To learn more about AoT compilation, visit [this section of Angular documentatio
 
 Prior to [Angular 2.2.0](https://github.com/angular/angular/blob/master/CHANGELOG.md#220-upgrade-firebooster-2016-11-14), `enableDebugTools()` would clobber `ng.probe`, which breaks Augury. Prior to that version, [this workaround](https://github.com/AngularClass/angular2-webpack-starter/blob/dbb7d10e6e84b8e88116d957f0047b422ab807c1/src/app/environment.ts#L28...L36) will circumvent the issue.
 
+## Support for Angular 9+
+
+Due to some Angular 9 debug API updates, Augury cannot support ngModule or router tree parsing for the moment. As a workaround, disable ivy to be able to see missing Augury tabs by adding this config to your project's `tsconfig.app.json` file : 
+```json
+"angularCompilerOptions": {
+  "enableIvy": false
+}
+```
+For more details about Ivy in Angular 9+, see [documentation](https://angular.io/guide/ivy#opting-out-of-ivy-in-version-9)
```

Comment:
This link specifically targets the section of ivy related to opting out. The text implies that the link should be related to Ivy in general.